io.github.jchempaint.JChemPaint.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.jchempaint.JChemPaint
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk8
+command: jchempaint
+
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk8/install.sh
+
+  - name: jchempaint
+    buildsystem: simple
+    build-options:
+      env:
+        PATH: /app/bin:/usr/bin:/usr/lib/sdk/openjdk8/bin
+        JAVA_HOME: /usr/lib/sdk/openjdk8/jvm/java-8-openjdk
+    build-commands:
+      - install -Dm755 jchempaint /app/bin/jchempaint
+      - install -Dm755 io.github.jchempaint.JChemPaint.desktop /app/share/applications/io.github.jchempaint.JChemPaint.desktop
+      - install -Dm755 io.github.jchempaint.JChemPaint.256.png /app/share/icons/hicolor/256x256/apps/io.github.jchempaint.JChemPaint.png
+      - install -Dm755 io.github.jchempaint.JChemPaint.metainfo.xml /app/share/metainfo/io.github.jchempaint.JChemPaint.metainfo.xml
+      - install -Dm644 JChemPaint.jar  /app/share/org.openscience/jchempaint/JChemPaint.jar
+    sources:
+      - type: file
+        url: https://github.com/JChemPaint/jchempaint/releases/download/3.4b/JChemPaint.jar
+        sha256: 34ed419e7b3a1a3eda1e934cf15dbb8c11a7beb216b79132f695993e488d3563
+        use-git-am: true
```

Comment:
what is this for? `use-git-am` There is no patch here.

Reply:
From the long tradition of ignoring flatpak-builder warnings:

```
** (flatpak-builder:2): WARNING **: 14:20:18.101: Unknown property use-git-am for type BuilderSourceFile
```

Reply:
It stems from https://github.com/JChemPaint/jchempaint/pull/205 which I at first tried to reference here, but it didn't work.